### ISSUE TITLE
use CLIF syntax for FOL definitions

### ISSUE THREAD

**Issue by wdduncan** [iofoundry/Core#5]
The FOL definitions contain symbols (e.g., the arrow "->" and existential backwards "E") that cannot be displayed properly when someone is editing the ontology is another modality (e.g., using a text editor). You should consider using CLIF (Common Logic Interchange Format) or [IKL](https://www.ihmc.us/users/phayes/IKL/GUIDE/GUIDE.html#TheLogic) syntax.

**Comment by schneidert** [comment]
I would disagree. CLIF notation is much more difficult to understand. First Order Logic notation is more commonly understood. The particular symbols used can be modified to avoid the display problems.

**Comment by wdduncan** [comment]
@schneidert Looks like we are in disagreement then :)  
I'm not wedded to CLIF. I proposed CLIF b/c the notation issues have already been worked out. If you have another FOL syntax to propose that would address such issues, then I would suggest that such documentation is posted as part of the repo/project in order to make it well known and explicit amongst the community.

EDIT:
P.S. I also suggested IKL. Does that suit you better?
